src/hooks/useEvent/useEvent.tsx

```diff
@@ -0,0 +1,13 @@
+import { useCallback, useLayoutEffect, useRef } from 'react'
+
+type AnyFunction = (...args: any[]) => any
+
+export function useEvent<T extends AnyFunction>(callback?: T) {
```

Comment:
please add comments here as to the original gist and the RFC as documentation

Reply:
Fixed ! thank you